### PR TITLE
Update Hangup-Cause.md

### PR DESCRIPTION
**Pull request by Astratus**
Improved/Fixed whitespace and formatting.

### CODE REVIEW

File: docs/Configuration/Miscellaneous/Hangup-Cause.md [hunk 1]
```diff
@@ -46,63 +46,59 @@ Note that in some cases, the hangup causes returned may not be reflected in . Fo
...
+```
 [default]
-
 exten => s,1,NoOp()
- same => n,Set(CHANNEL(hangup\_handler\_push)=handler,s,1)
- same => n,Dial(SIP/foo&SIP/bar,10)
- same => n,Hangup()
+same => n,Set(CHANNEL(hangup_handler_push)=handler,s,1)
+same => n,Dial(SIP/foo&SIP/bar,10)
+same => n,Hangup()
```
Comment (gtjoseph) [inline docs/Configuration/Miscellaneous/Hangup-Cause.md:54]: Thanks for fixing the code blocks and the backslashes.

Actually though, we tend to indent the `same` lines by a space so the `=>` line up...

```
exten => s,1,NoOp()
 same => n,Set(CHANNEL(hangup_handler_push)=handler,s,1)
 same => n,Dial(SIP/foo&SIP/bar,10)
 same => n,Hangup()
```

**Review by gtjoseph — CHANGES_REQUESTED** [review]

File: docs/Configuration/Miscellaneous/Hangup-Cause.md (hunk 1 above)
Comment (Astratus) [reply docs/Configuration/Miscellaneous/Hangup-Cause.md]: sames are now lined up.

**Review by Astratus — COMMENTED** [review]
sames now lined up.

**Comment by gtjoseph** [comment]
Can you squash the two commits into one and do a force push?

**Comment by Astratus** [comment]
> Can you squash the two commits into one and do a force push?

Done.

**Review by gtjoseph — APPROVED** [review]

**Review by jcolp — APPROVED** [review]

**Comment by Astratus** [comment]
Can we merge in this in now that it's approved and all checks have passed? Excited to see this go live.

**Comment by gtjoseph** [comment]
> Can we merge this in now that it's approved and all checks have passed? Excited to see this go live.

Yeah sorry.  I lost track of this.

**Comment by gtjoseph** [comment]
It should go live tonight when the publish job runs.

**Pull request merged**